packages/paste-core/components/form/src/shared/Prefix.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+
+const Prefix: React.FC = ({children}) => {
+  if (children == null) return null;
+  return (
+    <Box
+      display="flex"
+      alignItems="flex-start"
```

Comment:
I think if we remove this line we should get the same outcome with less output CSS

Reply:
We do need to set that here, other wise in certain cases the child element will span the full height so the click area is huge. See focus in screenshot:

![image](https://user-images.githubusercontent.com/1350081/76781395-43d32600-6774-11ea-8c93-be80a4e19526.png)